packages/api/pages/api/languages/[code]/verses/[verseId]/notes.ts

```diff
@@ -35,9 +35,12 @@ export default createRoute<{ code: string; verseId: string }>()
           COALESCE(fn."content", '') AS "footnoteContent"
         FROM "Word" AS w
 
-        LEFT JOIN "PhraseWord" AS phw ON phw."wordId" = w.id
-        LEFT JOIN "Phrase" AS ph ON ph.id = phw."phraseId"
-          AND ph."languageId" = ${language.id}::uuid
+        LEFT JOIN LATERAL (
+          SELECT phw."wordId", ph.id FROM "PhraseWord" AS phw
```

Comment:
We are only using the phrase id once this is joined, so we can remove the word id from the select here
```suggestion
          SELECT ph.id FROM "PhraseWord" AS phw
```